os/mm/mm_heap/mm_memalign.c

```diff
@@ -56,22 +56,42 @@
 
 #include <tinyara/config.h>
 
-#include <assert.h>
+#include <debug.h>
 
 #include <tinyara/mm/mm.h>
 
+#ifdef CONFIG_DEBUG_MM_HEAPINFO
+#include  <tinyara/sched.h>
+#endif
+#include "mm_node.h"
+
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
 
+#ifndef NULL
+#define NULL ((void*)0)
```

Comment:
Please add space between `void` and `*`.

Reply:
This is not required. Removed it.